File: 1470-shuffle-the-array/1470-shuffle-the-array.go
func shuffle(nums []int, n int) []int {
    res := make([]int, 0)
    for i:=0;i<len(nums)-n;i++{
        for j:=i;j<len(nums);j+=n{
            res = append(res, nums[j])
        }
    }
    return res
}

File: 1470-shuffle-the-array/1470-shuffle-the-array.py
class Solution:
    def shuffle(self, nums: List[int], n: int) -> List[int]:
        # res = []
        x = nums[:n]
        y = nums[n:]
        fx, fy = 0, 0
        for i in range(len(nums)):
            if i%2 == 0:
                nums[i] = x[fx]
                fx += 1
            else:
                nums[i] = y[fy]
                fy += 1
        return nums